invokeai/backend/generator/base.py

```diff
@@ -181,8 +182,9 @@ def load_generator(self, model: StableDiffusionGeneratorPipeline, generator_clas
         return generator_class(model, self.params.precision)
 
     def get_scheduler(self, scheduler_name:str, model: StableDiffusionGeneratorPipeline)->Scheduler:
-        scheduler_class = self.scheduler_map.get(scheduler_name,'ddim')
-        scheduler = scheduler_class.from_config(model.scheduler.config)
+        scheduler_class, scheduler_extra_config = self.scheduler_map.get(scheduler_name,'ddim')
```

Comment:
Same as in previous comment - second argument is default value, not default key.